### PR TITLE
DBZ-573 Geometry documentation

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-573

### CODE REVIEW

File: docs/connectors/postgresql.asciidoc [hunk 1]
```diff
@@ -965,19 +965,31 @@ PostgreSQL supports `NaN` (not a number) special value to be stored in the `DECI
...
+|`STRUCT`
+|`io.debezium.data.geometry.Geometry`
+|Contains a structure with 2 fields +
 
-|`POINT`
+* `srid (INT32)` - Spatial Reference System Identifier defining what type of geometry object is stored in the structure
+* `wkb (BYTES)` - a binary representation of the geometry object encoded in thw Well-Known-Binary format.
+Please see http://www.opengeospatial.org/standards/sfa[Open Geospatial Consortium Simple Features Access specification] for the format details.
+
+|`GEOGRAPHY` (spherical)
```
Comment (gunnarmorling) [inline docs/connectors/postgresql.asciidoc]: Isn't `GEOMETRY` missing?

File: docs/connectors/postgresql.asciidoc [hunk 2]
```diff
@@ -965,19 +965,31 @@ PostgreSQL supports `NaN` (not a number) special value to be stored in the `DECI
...
 |Semantic type (schema name)
 |Notes
+|planar types, e.g. `POLYGON`, `MULTIPOINT`
+|`STRUCT`
+|`io.debezium.data.geometry.Geometry`
+|Contains a structure with 2 fields +
 
-|`POINT`
+* `srid (INT32)` - Spatial Reference System Identifier defining what type of geometry object is stored in the structure
+* `wkb (BYTES)` - a binary representation of the geometry object encoded in thw Well-Known-Binary format.
```
Comment (gunnarmorling) [inline docs/connectors/postgresql.asciidoc]: s/thw/the/

File: docs/connectors/postgresql.asciidoc (hunk 1 above)
Comment (jpechane) [reply docs/connectors/postgresql.asciidoc]: @gunnarmorling Hmm, it seems that push witout commit is not the way how git works ;-)

**Comment by gunnarmorling** [comment]
Ahaha :)

**Comment by gunnarmorling** [comment]
Thanks!

**Pull request merged**